fendermint/vm/interpreter/src/fvm/gas_estimation.rs

```diff
@@ -10,9 +10,10 @@ use crate::fvm::{
 use fendermint_vm_message::query::GasEstimate;
 use fvm_ipld_blockstore::Blockstore;
 use fvm_ipld_encoding::{self, RawBytes};
-use fvm_shared::{
-    bigint::BigInt, econ::TokenAmount, error::ExitCode, message::Message, BLOCK_GAS_LIMIT,
-};
+use fvm_shared::{bigint::BigInt, econ::TokenAmount, error::ExitCode, message::Message};
+
+// BLOCK_GAS_LIMIT was removed in FVM 4.7, define locally for IPC
+const BLOCK_GAS_LIMIT: u64 = 10_000_000_000;
```

Comment:
that means FVM 4.7 no longer enforce block gas? Does this mean ipc should increase block gas limit?